src/types/export-status.ts

```diff
@@ -0,0 +1,4 @@
+export enum ExportStatus {
+  Exporting = 'exporting',
```

Comment:
```suggestion
  EXPORTING = 'exporting',
```

we use uppercase letters for enum values